src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3050,12 +3050,13 @@ public override IEnumerable<ISymbol> GetSymbolsWithName(Func<string, bool> predi
             return new PredicateSymbolSearcher(this, filter, predicate, cancellationToken).GetSymbolsWithName();
         }
 
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
these overloads shouldnot be a problem.  There is no conflict/ambiguity between them and the existing overloads that take delegates.